gr-fec/include/gnuradio/fec/awgn_bp.h

```diff
@@ -36,14 +36,14 @@ class FEC_API awgn_bp
     //! Default constructor
     awgn_bp(){};
 
-    //! A constructor for given GF2Mat and sigma
-    awgn_bp(const GF2Mat X, float sgma);
```

Comment:
API change

Reply:
The new LDPC algorithm does not have sigma as a parameter, therefore it is unused and unnecessary. I would also say it never should've been there in the first place since the soft decisions should already take this into account.

Reply:
What I can say here is that the previous LDPC decoder which had a `sigma` was technically broken for anything besides BPSK and QPSK. It's not possible to compute LLRs correctly without a knowledge of the constellation. The old LDPC decoder always computed LLRs by using the BPSK/QPSK formula, so it wasn't possible to use it correctly with other constellations. In the new LDPC decoder, the task to compute LLRs has been moved to the constellation soft decoder (and for BPSK/QPSK the LLR can also be computed with a multiply).

I can't think of a way to fix this brokeness without an API change.